tools/hello_world.dart

```diff
@@ -0,0 +1,23 @@
+#!tools/find_dart.sh
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Note that this file doesn't live in a "traditional" pub package.
```

Comment:
As a mechanism to avoid the overhead of adding new Dart code in the engine repo, I think this is great. But I'd still like to see us follow conventions around separating code into packages with (maybe most importantly) `test/` subdirectories, for example.